docs/02-guides/server-side-events/integrations/google-analytics.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Google Analytics
+description:
+  Implement server-side events with Google Analytics Measurement Protocol.
+---
+
+<p>{frontMatter.description}</p>
+
+## Introduction
+
+Google Analytics provides a JS SDK to push events to their server. To do
+server-to-server communication, you need to use Google Analytics Measurement
+Protocol.
+
+## Setup Google Analytics
+
+To use Google Analytics Measurement Protocol, you need to get your API Secret
+and your measurement ID. Please follow the
+[Google Analytics Measurement Protocol Documentation](https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference?client_type=gtag#payload_query_parameters)
+to learn how to get them.
```

Comment:
I think it would be better to follow the same format as you did for addingwell, with the prerequisites, see https://github.com/front-commerce/developers.front-commerce.com/pull/903/files#diff-c1aac838907413287ce66b59bb70a572bfcfced268014fe0f40c5b0e156147bbR9-R16